README.rst

```diff
@@ -17,6 +17,20 @@ pytest-twisted is a plugin for pytest, which allows to test code,
 which uses the twisted framework. test functions can return Deferred
 objects and pytest will wait for their completion with this plugin.
 
+
+Python 2 support plans
+==================
```

Comment:
I believe the underlying header needs to be at least the size of the title.

Text seems fine, thanks for writing it. 👍